stories/General/IconStory.tsx

```diff
@@ -110,6 +137,32 @@ const LikeButton = styled(ThumbsUpOutlineIcon)\`
           ))}
       </div>
     </StorybookComponent>
+    <StorybookComponent
+      title="Icon Sizes"
+      componentDescription="Icons are auto-scaled to provide multiple sizes. These sizes are used depending on the usage."
+      usage={` <StarIcon size='xxs' />
+      <StarIcon size='xs' />
+      <StarIcon size='s' />
+      <StarIcon size='m' />
+      <StarIcon size='l' />
+      <StarIcon size='xl' />
+      <StarIcon size='xxl' />
+      <StarIcon size='xxxl' />
+      `}
+    >
+      <div style={{ display: 'flex', flexWrap: 'wrap' }}>
```

Comment:
would like to suggest adding `align-items: flex-end;` style here to make Icon examples looks tidy. 

like
![image](https://user-images.githubusercontent.com/13249748/95695172-47764f00-0c68-11eb-88e1-ea36c70bd989.png)